### DISCUSSION TITLE
v1.5.9-00

### DISCUSSION THREAD

**Discussion by gaborauth** [gaborauth/toolsboox-android#106] [Announcements]
- Added multiple notes option to the every calendar view.
- Added multiple dots per note page as marks.
- Added trash icon to erase all content of the page.
- Replaced "back" icon with "calendar" icon.
- Updated Spanish localization.
- Small bugfixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/gaborauth/toolsboox-android/releases/tag/v1.5.9-00'>v1.5.9-00</a>.</em>

**Comment by pjmysle** [comment]
Tried to install a few times. Receive App Not Installed message. Tried downloading again, turning off NetGuard, restarting. No change.

**Reply by gaborauth** [reply]
Strange.

Could it be that you installed it from Google Play before? The two applications from the two source are different.

**Reply by pjmysle** [reply]
You got it right. Thanks! I uninstalled and reinstalled and it works fine. Nice that it retained my data. I'll test out the new features. Looks good!

**Reply by gaborauth** [reply]
> Nice that it retained my data.

It's only on Android 10 and bellow, with Android 11+ the uninstall removes the application data too!

**Comment by pjmysle** [comment]
Upgraded Boox firmware to latest 3.3.1 last night, and using v.1.5.9.  The refresh while in Tools for Boox seems to be negatively affected. Very disruptive refresh while making notes. Approximately every 15 secs while in Normal mode and more frequent in Regal mode.

**Reply by gaborauth** [reply]
Okay, I will update to 3.3.1 too in this week and try to fix it!
